src/OpenClaw.Dashboard/wwwroot/css/app.css

```
@@ -0,0 +1,490 @@
+    --gap-1: 4px;
+    --gap-2: 8px;
+    --gap-3: 12px;
+    --gap-4: 16px;
+    --gap-5: 24px;
+    --gap-6: 32px;
+    --gap-7: 48px;
+
+    /* ── Type ──────────────────────────────────────────────────────────────── */
+    --font-display: "Bricolage Grotesque", "Manrope", system-ui, sans-serif;
+    --font-body:    "Manrope", "Bricolage Grotesque", system-ui, sans-serif;
+    --font-mono:    "JetBrains Mono", ui-monospace, "SF Mono", Menlo, monospace;
```

Stylelint (17.11.1)

[error] Expected "Menlo" to be "menlo" (value-keyword-case)

(value-keyword-case)